tests/testthat/test-recall_count.R

```diff
@@ -0,0 +1,31 @@
+test_that("count_recall_by_make returns expected value", {
+    query <- count_recall_by_make("Nissan", start_year = 1995, end_year = 1998)
+    expect_type(query, "list")
+    expect_equal(query[[1]], 43L)
+    expect_equal(nrow(query), 1)
+    expect_equal(colnames(query), "Result Count")
+})
+
+test_that("count_recall_by_make with manufacturer", {
+    query <- count_recall_by_make("Nissan", manufacturer = TRUE, start_year = 1995, end_year = 1998)
+    expect_type(query, "list")
+    expect_equal(query[[1]], 59L)
+    expect_equal(nrow(query), 1)
+    expect_equal(colnames(query), "Result Count")
+})
+
+test_that("count_recall_by_model returns expected value", {
+    query <- count_recall_by_model("Altima", start_year = 1995, end_year = 1998)
+    expect_type(query, "list")
+    expect_equal(query[[1]], 2L)
+    expect_equal(nrow(query), 1)
+    expect_equal(colnames(query), "Result Count")
+})
+
+test_that("count_recall_by_years returns expected value", {
+    query <- count_recall_by_years(start_year = 1995, end_year = 1998)
+    expect_type(query, "list")
+    expect_equal(query[[1]], 5597L)
+    expect_equal(nrow(query), 1)
+    expect_equal(colnames(query), "Result Count")
+})
```

Comment:
should maybe have another call where no years are entered `count_recall_by_years()`

Reply:
I did have that initially.  The only real difference is that I can't expect to know what the count number will be since this database is being updated and our null values should include the entire database. I just had that it should expect and integer

Reply:
probably worth adding and then just testing that it does return an int